### PR TITLE
fix(interactive): global selected event emit

### LINKED ISSUE #899: 🐛内置交互：S2Event.GLOBAL_SELECTED没有被增量选择以及快捷框选触发
### 🏷 Version

| Package      | Version |
| -------------- | --------- |
| @antv/s2       | -  1.3.0    |
| @antv/s2-react | -  1.3.0       |

### 🖋 Description

获取表格实例后订阅S2Event.GLOBAL_SELECTED事件，command增量选择、shift快捷选择无法触发事件
### ⌨️ Code Snapshots

nope
### 🤔 Steps to Reproduce

同描述
### 💻 System information

chrome: v96, MacOs:10.14

### PR DESCRIPTION
**Pull request by YardWill**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [x] Solve the issue and close #899 

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/__tests__/unit/interaction/shift-multi-selection-spec.ts [hunk 1]
```diff
@@ -121,6 +121,10 @@ describe('Interaction Shift Multi Selection Tests', () => {
 
     s2.getCell = () => mockCell11.mockCell as any;
 
+    s2.on(S2Event.GLOBAL_SELECTED, (cells) => {
+      expect(cells).toEqual([]);
```
Comment (lijinke666) [inline packages/s2-core/__tests__/unit/interaction/shift-multi-selection-spec.ts:125]: 用 jest.fn 去做测试吧，参考其他，这样写，如果事件没触发，断言也是通过的，而且，不应该是空数组吧

Comment (lijinke666) [reply packages/s2-core/__tests__/unit/interaction/shift-multi-selection-spec.ts:125]: image

**Comment by lijinke666** [comment]
packages/s2-core/__tests__/unit/interaction/data-cell-multi-selection-spec.ts 也需要加一个

**Comment by YardWill** [comment]
> packages/s2-core/**tests**/unit/interaction/data-cell-multi-selection-spec.ts 也需要加一个

加了

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**